data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report3_col_suppression.py

```diff
@@ -178,85 +155,62 @@
 SUM(CASE WHEN provider_id IS NOT NULL THEN 1 ELSE 0 END) AS n_provider_id_not_null
 FROM `{{project_id}}.{{deid_cdr}}.condition_occurrence`
 """)
-q = query.render(project_id=project_id, deid_cdr=deid_cdr)
-df1 = execute(client, q)
-if df1.loc[0].sum() == 0:
-    df = df.append({
-        'query': 'Query5 Condition provider_id',
-        'result': 'PASS'
-    },
-                   ignore_index=True)
+q = query.render(project_id=project_id,deid_cdr=deid_cdr)  
+df1=execute(client, q) 
+if df1.loc[0].sum()==0:
+ df = df.append({'query' : 'Query5 Condition provider_id', 'result' : 'PASS'},  
+                ignore_index = True) 
 else:
-    df = df.append(
-        {
-            'query': 'Query5 Condition provider_id',
-            'result': 'Failure'
-        },
-        ignore_index=True)
-df1
+ df = df.append({'query' : 'Query5 Condition provider_id', 'result' :  'Failure'},  
+                ignore_index = True) 
+df1        
 
-# # 6 Verify the following columns in the DEVICE_EXPOSURE table have been set to null:
+# # 6 Verify the provider_id in the DEVICE_EXPOSURE table is null and visit_detail_id is not all null:
 #
-# <font color='red'> Uncomment below when visit_detail_id is available
+# <font color='red'> DST-1423,updates, only provide_id is null.
```

Comment:
Suggested change. Would you change this comment to something like:
```
<font color='red'> 'DST-1423, updates. Verify that provider_id is suppressed(null) and that visit_detail_id is not suppressed(not always null) in `device_exposure`'. <br>
    
`n_provider_id_not_null` should return 0 <br>
`n_visit_detail_id_not_null` should return > 0  <br>
```

When we run these notebooks and we see results with a count it usually means the check is failing. We don't always go down to the summary to figure out if the check is failing. I think this comment will clear up any possible confusion.